### ISSUE TITLE
Newspaper always a Periodical?

### ISSUE THREAD

**Issue by TobiasNx** [hbz/lobid-resources#1991]
Is newspaper a subset of Periodical therefore all Newspaper also have type Periodical?

I was stumbling over old and new records in ALMA/ALEPH, e.g.:
http://lobid.org/resources/990117329470206441.json

**Comment by TobiasNx** [comment]
From closed: #1993 

http://lobid.org/resources/990054089950206441
http://aleph.lobid.org/resources/HT001387709

In Aleph Spiegel used to be a type:Periodical and a type:Newspaper. Now it is only a Newspaper.
They seem not to be mutual exclusive but are Newspaper a subset of Periodicals and therefore all Newspapers are Periodicals too?

**Comment by TobiasNx** [comment]
Looks good:

```
{
  "@context" : "http://lobid.org/resources/context.jsonld",
  "id" : "http://lobid.org/resources/990117329470206441#!",
  "type" : [ "BibliographicResource", "Newspaper", "Periodical" ],
  "medium" : [ {
```

http://lobid.org/resources/990117329470206441.json
